pkg/controller/operators/catalogtempate/operator.go

```diff
@@ -0,0 +1,319 @@
+package catalogtempate
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/informers/externalversions"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/catalogsource"
```

Comment:
Thanks... I learned something today. I always let the auto format handle imports thinking it would take care of the ordering, but I see now that I can use returns to keep blocks organized.

Reply:
Also I assume that "everything else" includes the `api` project since its not part of the `operator-lifecycle-manager`

Reply:
fixed assuming I understand the organization, this has been fixed